responsibleai/tests/causal/conftest.py

```diff
@@ -8,7 +8,9 @@
 import pytest
 import shap
 from sklearn.model_selection import train_test_split
-from tests.common_utils import create_adult_income_dataset, create_housing_data
+from tests.common_utils import create_adult_income_dataset
```

Comment:
curious we still don't have create_adult_income_dataset in rai_test_utils?

Reply:
Because we would need to depend on dice-ml in rai_test_utils. rai_test_utils is a simple package and I don't want complex dependencies to be present in it.